### PR TITLE
[RC2] Updated client API - Similiar job execution as in PECL gearman extension

### PR DESCRIPTION
**Pull request by mhlavac**
@Publero/developers

### CODE REVIEW

File: lib/Net/Gearman/Client.php [hunk 1]
```diff
@@ -134,25 +132,127 @@ protected function getConnection()
...
+        return $this->runSingleTaskSet($this->createSet($functionName, $workload, $unique, Task::JOB_LOW));
+    }
+
+    /**
+     * @param string $functionName
+     * @param string $workload
+     * @param string $unique
+     * @return string
+     */
+    protected function runSingleTaskSet($set)
```
Comment (tomas-pecserke) [inline lib/Net/Gearman/Client.php]: wrong phpdoc

File: lib/Net/Gearman/Client.php [hunk 2]
```diff
@@ -134,25 +132,127 @@ protected function getConnection()
...
+     * @param string $functionName
+     * @param string $workload
+     * @param string $unique
+     * @return Set
+     */
+    private function createSet($functionName, $workload, $unique = null, $type = Task::JOB_NORMAL)
     {
         if (null === $unique) {
-            $unique = getmypid() . '_' . uniqid();
+            $unique = getmypid()."_".uniqid();
```
Comment (tomas-pecserke) [inline lib/Net/Gearman/Client.php]: medzery opkolo `.`

**Comment by tomas-pecserke** [comment]
:see_no_evil:

**Comment by mhlavac** [comment]
#### RC2

@Publero/developers

**Comment by tomas-pecserke** [comment]
# ok

**Pull request merged**